src/core/keyboard/listener.coffee

```diff
@@ -72,3 +73,8 @@ module.exports = class KDKeyboardListener
     @currentListener = new this
     @currentListener.listen()
     return @currentListener
+
+  @record = (fn) ->
```

Comment:
`@record = (fn) -> Mousetrap.record fn.bind fn` is better if you don't require the explicit `undefined` return.